### PR TITLE
[INJICERT-657] add dev1 landregistry cache config

### PR DESCRIPTION
**Pull request by vharsh**

### CODE REVIEW

File: certify-postgres-landregistry.properties [hunk 1]
```diff
@@ -25,6 +36,16 @@ mosip.certify.data-provider-plugin.postgres.scope-query-mapping={\
...
 
+## ------------------------------------------- Cache specific usecase properties ------------------------------------------------------------
+# Velocity Template caching for performance of VCI -- required when Certify is deployed in plugin-mode=DataProvider
+mosip.certify.cache.names=templatecache
+spring.cache.cache-names=${mosip.certify.cache.names}
+# going with a modest size considering no multi-tenancy
+mosip.certify.cache.size={templatecache': 20}
+# cache duration set to half-day
+mosip.certify.templatecache-expire-seconds=43200
+mosip.certify.cache.expire-in-seconds={'templatecache': ${mosip.certify.templatecache-expire-seconds}}
```
Comment (vishwa-vyom) [inline certify-postgres-landregistry.properties:47]: Can we not keep the cache section in the certify-default? Normally usecase specific, they will not change these much

File: certify-postgres-landregistry.properties [hunk 2]
```diff
@@ -2,21 +2,32 @@
...
 ## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
 ## Postgres DB specific config
 
 # Other supported value: [RsaSignature2018]
 mosip.certify.data-provider-plugin.issuer.vc-sign-algo=Ed25519Signature2020
 mosip.certify.vcformat.vc.expiry=true
-mosip.certify.data-provider-plugin.issuer-public-key-uri=did:web:vharsh.github.io:DID:devcp2#key-0
-mosip.certify.data-provider-plugin.issuer-uri=did:web:vharsh.github.io:DID:devcp2
+mosip.certify.data-provider-plugin.issuer-public-key-uri=did:web:inji-config.github.io:dev1:postgreslandregistry#key-0
+mosip.certify.data-provider-plugin.issuer-uri=did:web:inji-config.github.io:dev1:postgreslandregistry
```
Comment (vishwa-vyom) [inline certify-postgres-landregistry.properties:27]: can we keep this to the namespace ? like did:web:inji-config.github.io:dev1:cp1 etc.. ?
Also can you send the PR to make the change in gh pages branch also ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-postgres-landregistry.properties (hunk 1 above)
Comment (vharsh) [reply certify-postgres-landregistry.properties:47]: The `templatecache` is required only by `DataProvider` plugin and not the other plugin implementations such as `mosipid`, `insurance`, `mock`, etc.

Comment (vishwa-vyom) [reply certify-postgres-landregistry.properties:47]: that is ok, this config ideally will not create any issues right ?

File: certify-postgres-landregistry.properties [hunk 3]
```diff
@@ -2,21 +2,32 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.certify.cp2.security.pin}
```
Comment (vishwa-vyom) [inline certify-postgres-landregistry.properties:5]: this is a dev1 setup related change, we cannot commit this into release branch.

File: certify-postgres-landregistry.properties (hunk 2 above)
Comment (vishwa-vyom) [reply certify-postgres-landregistry.properties:27]: ??

File: certify-postgres-landregistry.properties [hunk 4]
```diff
@@ -25,6 +36,16 @@ mosip.certify.data-provider-plugin.postgres.scope-query-mapping={\
...
 
 
+## ------------------------------------------- Cache specific usecase properties ------------------------------------------------------------
+# Velocity Template caching for performance of VCI -- required when Certify is deployed in plugin-mode=DataProvider
+mosip.certify.cache.names=templatecache
+spring.cache.cache-names=${mosip.certify.cache.names}
+# going with a modest size considering no multi-tenancy
+mosip.certify.cache.size={templatecache': 20}
+# cache duration set to half-day
+mosip.certify.templatecache-expire-seconds=43200
```
Comment (vishwa-vyom) [inline certify-postgres-landregistry.properties:46]: Lets make this 30 min

File: certify-postgres-landregistry.properties (hunk 2 above)
Comment (vishwa-vyom) [reply certify-postgres-landregistry.properties:27]: remove postgres keep only landregistry

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-mock-mdl.properties [hunk 5]
```diff
@@ -19,6 +19,8 @@ mosip.certify.domain.url=https://${mosip.injicertify.mock.host}
 mosip.certify.identifier=${mosip.certify.domain.url}
 mosip.certify.authorization.url=https://${mosip.esignet.mock.host}
 mosip.certify.database.name=inji_certify_mock
+# conditionally set the SOFTHSM pin based on the setup requirements and fallback to the latter if former value is missing
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.certify.mockid.security.pin:${softhsm.certify.security.pin}}
```
Comment (vharsh) [inline certify-mock-mdl.properties:23]: As of now, the mockid security PIN is going to be the same for all ENVs, later when we deploy me might plan to 

1. change the way how it's done
2. make certain exceptions to the rule to make the deployment simpler, i.e. deploy inji-certify-mock-mdl in the same namespace as inji-certify-mock pod


cc @abhishek8shankar @vishwa-vyom

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**